### ISSUE TITLE
aarch64-linux bindist requires libnuma

### ISSUE THREAD

**Issue by mpickering** [haskell/ghcup-hs#761]
The aarch64-linux bindist appears to be built with a version of GHC where the RTS depends on libnuma. This means that on aarch64-linux that libnuma has to be installed for ghcup to work.

This is probably a packaging mistake from the 8.10.7 series and will probably be fixed by building the bindist with 9.2 or 9.4 bindists.

**Comment by hasufell** [comment]
Yes. This is a bug that had been fixed in GHC and then regressed. Is there a list of GHC versions/bindists that are affected?

**Comment by mpickering** [comment]
I fixed it in this commit: https://gitlab.haskell.org/ghc/ghc/-/merge_requests/7357

Which was backported to 9.2 series. So 9.2.2 onwards should be fine.

**Comment by hasufell** [comment]
That doesn't seem to be the case. The binary has been build with GHC-9.2.5. Here is proof: https://downloads.haskell.org/~ghcup/0.1.19.0/aarch64-linux-ghcup.plan.json

**Comment by mpickering** [comment]
The error I see is:

```
cabal: error while loading shared libraries: libnuma.so.1: cannot open shared object file: No such file or directory
```

So it is actually the cabal binary which is built with 8.10.7 perhaps?

**Comment by hasufell** [comment]
cabal-install was built with 8.10.7 indeed: https://gitlab.haskell.org/haskell/cabal/-/jobs/1073137
